MaxText/configs/base.yml

```diff
@@ -225,3 +225,7 @@ decode_sampling_strategy: "greedy" # decode_sampling_strategy should be one of g
 decode_sampling_nucleus_p: -1 # set if you're doing nucleus / top-p
 decode_sampling_top_k: 0 # set if you're doing top-k
 decode_sampling_temperature: 1.
+
+# enabling slurm for multiprocess in gpus
+# It must be turned on while using SLURM
+multiprocess_gpu: False
```

Comment:
Sincerely sorry for the trouble. But let's use:
https://github.com/google/maxtext/blob/main/MaxText/configs/base.yml#L99

Reply:
Sure, I can add appropriate args to the following flag